src/components/CardGrid/CardGrid.js

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from 'react';
+import useStore from '../../hooks/useStore';
+import StyledFlex from '../../../styles/StyledFlex';
+import StyledDiv from '../../../styles/StyledDiv';
+import Typography from '../../../styles/Typography';
+import SVGIcon from '../../assets/SVGIcon/SVGIcon';
+
+export default function Card() {
+	const myVehicles = useStore(state => state.myVehicles);
+	const [mounted, setMounted] = useState(false);
+
+	useEffect(() => {
+		setMounted(true);
+	}, []);
```

Comment:
I am not sure about this, but since you are setting `mounted` to `true` on first render, wouldn't it be the same as saying 
`const [mounted, setMounted] = useState(true);`
without the useEffect?

Reply:
der setMounted muss ja aber gesetzt werden für den useState. Wo würdest du den anstelle platzieren?

Reply:
If you initialize the state as stated above with `const [mounted, setMounted] = useState(true);`, the value of `mounted` is `true` and you don't need `setMounted` and the useEffect during the rendering phase.

Besides, you could also destructure the state without the set function, as `const [mounted] = useState(true);`

However, if you don't ever change the `mounted` state, why do you need the state at all?

Reply:
I need it because otherwise I run into a hydration problem:
Error: Hydration failed because the initial UI does not match what was rendered on the server.

If I destruct as you recommended I run into the error of "useState Call is not destructured into  value + setter pair".

Hence I would like to keep it as is.

Reply:
solved via dynamic import with no SSR